mlflow/tracking/_tracking_service/client.py

```diff
@@ -489,6 +489,9 @@ def log_batch(
             represents future for logging operation.
 
         """
+        from mlflow.tracking.fluent import get_active_model_id
+
+        active_model_id = get_active_model_id()
```

Comment:
Do we need this?

Reply:
This is log_batch, I don't want to iterate over metrics in client's log_batch so updating here